### ISSUE TITLE
[AndroidTV, tvOS] - Cannot seem to set AppIcon

### ISSUE THREAD

**Issue by iAMkVIN-S** [flexn-io/renative#1745]
Good afternoon,

Seemingly app icons should be set by using the appConfig and renative.json. However, when looking through the documentation and schema for the renative.json file, I cannot for the life of me find where to set the app icons (other than replacing the png files directly under `platformBuilds/app_androidtv/...` and `platformBuilds/app_tvos`) - but I assume these are generated folders that should not be touched.

Looked through:
- renative.json schema
- website
- github issues
- online / chatGPT

Please advise 🙏

TLDR:
**How do we set App Icons for Android TV and tvOS in Renative?**

**Comment by ElenaDiachenko** [comment]
@iAMkVIN-S Hello. You can set app icons or any other resources by creating an assets folder in appConfigs/app with the assets required for those platforms. 
<img width="342" alt="assets" src="https://github.com/user-attachments/assets/d41a2868-2e93-42bd-9dda-5c7b601f8c86">


In renative.json this can be configured if needed

```   "platforms": {
        "androidtv": {
           "templateAndroid": {
            "AndroidManifest_xml": {
                "tag": "manifest",
                "children": [
                    {
                        "tag": "application",
                        "android:name": ".MainApplication",
                        "android:icon": "@mipmap/ic_launcher",
                        "android:banner": "@drawable/banner"
                    }
                ]
            }
           }
        },
        "tvos": {
            "templateXcode": {
                "project_pbxproj": {
                    "resourceFiles": []
                }
            }
        }
    }
```

**Comment by iAMkVIN-S** [comment]
Hello @ElenaDiachenko thank you so much for the quick reply!

I have setup my folder structure with icons and renative.json, but when I build I still have the default renative icons in the `platformBuilds/app_androidtv/app/src/main/res`.

I ran:
`npx rnv clean` (to completely delete my `platformBuilds`)
`npx rnv build -p androidtv -s release -r`

**Renative JSON:**
<img width="481" alt="image" src="https://github.com/user-attachments/assets/65330694-6db7-43ea-9e06-e03520b9c7ee">

**File Structure:** 
<img width="418" alt="image" src="https://github.com/user-attachments/assets/1b0a227c-7f1b-4dcd-9a2e-3f35921ed0a8">

**Comment by ElenaDiachenko** [comment]
@iAMkVIN-S  For now you can change the folder name from `androidtv` to `android` in AppConfigs/app/assets. The default settings for androidtv include
```
 "assetFolderPlatform": "android",
```
which is useful when you want to use the same android assets in androidtv and want to avoid asset duplication. 

Thanks for reporting this
